src/v2/guide/migration.md

```diff
@@ -1131,25 +1131,26 @@ You may notice that:
 
 ## Slots
 
-### Duplicate Slots <sup>removed</sup>
+### Slots Duplicados <sup>removido</sup>
 
-It is no longer supported to have `<slot>`s with the same name in the same template. When a slot is rendered it is "used up" and cannot be rendered elsewhere in the same render tree. If you must render the same content in multiple places, pass that content as a prop.
+Ya no se admite tener `<slot>`s con el mismo nombre en el mismo template. Cuando se renderiza un slot, se "agota" y no se puede procesar en otro lugar del mismo árbol de procesamiento. Si debe representar el mismo contenido en varios lugares, pase ese contenido como propiedad.
```

Comment:
Aren't we using `plantilla` instead of leaving `template`?
```suggestion
Ya no se admite tener `<slot>`s con el mismo nombre en la misma plantilla. Cuando se renderiza un slot, se "agota" y no se puede procesar en otro lugar del mismo árbol de procesamiento. Si debe representar el mismo contenido en varios lugares, pase ese contenido como propiedad.
```

Reply:
Yes, you're right. Sometimes I enter a loop that I can't avoid, and I start thinking what was the decision que made regarding some translations. 

By the way, I just created the ticket: https://trello.com/c/HLLpAX87 to place this word mappings there :)